### PR TITLE
use fork instead of cluster

### PR DESCRIPTION
**Pull request by bkniffler**
As referred to in:

https://github.com/ericclemmons/start-server-webpack-plugin/issues/40
https://github.com/ericclemmons/start-server-webpack-plugin/issues/12

### CODE REVIEW

**Comment by bkniffler** [comment]
I'll check if we can use child_process directly, just used execa to create a POC.

**Comment by bkniffler** [comment]
Okay, fork works great in my case.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "run-script-webpack-plugin",
+  "name": "run-script-webpack-plugin-fork",
```
Comment (atassis) [inline package.json:2]: Shouldn't rename

File: package.json [hunk 2]
```diff
@@ -33,12 +33,13 @@
...
+    "webpack": "5.*.*",
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
     "eslint": "^7.14.0",
     "eslint-config-prettier": "^6.15.0",
     "prettier": "^2.2.1",
     "typescript": "^4.1.2"
-  }
+  },
+  "dependencies": {}
```
Comment (atassis) [inline package.json:44]: No need in adding this

File: package.json [hunk 3]
```diff
@@ -33,12 +33,13 @@
   "homepage": "https://github.com/atassis/run-script-webpack-plugin#readme",
   "devDependencies": {
     "@types/node": "^14.14.10",
-    "@types/webpack": "^4.41.25",
+    "webpack": "5.*.*",
```
Comment (atassis) [inline package.json:36]: wildcards? why?

**Review by atassis — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (bkniffler) [reply package.json:2]: Woops, its a remain of me, had to publish it urgently for my own need :)

File: package.json (hunk 2 above)
Comment (bkniffler) [reply package.json:44]: Yep

File: package.json (hunk 3 above)
Comment (bkniffler) [reply package.json]: Changed it to "5" simply, according to https://docs.npmjs.com/about-semantic-versioning it has the same effect. If you have a preference, let me know.

Comment (atassis) [reply package.json]: I prefer to clearly set the version with caret sign `^` so it'ld gonna show the last version tested

Comment (bkniffler) [reply package.json]: Okay, done

**Review by atassis — APPROVED** [review]

**Pull request merged**